Makefile

```diff
@@ -181,14 +181,15 @@ NIM_PARAMS += --outdir:./bin -d:debug
 CFLAGS += -g
 CXXFLAGS += -g
 
+# TODO: control debug/release builds with a Make var
```

Comment:
@michaelsbradleyjr ,  Would creating a variable that by default assumes it's a Debug build work here? I want to enable QML debugging only on DOtherside `Debug` builds.  (I think this could be useful too for the nim param on line 179)